<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApkReport</class>
 <widget class="QDialog" name="ApkReport">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>282</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>282</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>282</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>APK Existance Report</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>230</width>
     <height>241</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lMissed">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Missed:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lDouble">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Posible doble apks:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lUnused">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Unused apks:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lLoading">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lCantM">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color: rgba(0, 0, 0, 150);
border: 5px; border-radius:20px;
color:white;
font: bold 10pt &quot;MS Shell Dlg 2&quot;;}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lCantD">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color: rgba(0, 0, 0, 150);
border: 5px; border-radius:20px;
color:white;
font: bold 10pt &quot;MS Shell Dlg 2&quot;;}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lCantU">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
background-color: rgba(0, 0, 0, 150);
border: 5px; border-radius:20px;
color:white;
font: bold 10pt &quot;MS Shell Dlg 2&quot;;}</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="tbRefresh">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Refresh (F5)</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="check.qrc">
     <normaloff>:/img/refresh.png</normaloff>:/img/refresh.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Send Selected Unused to Process (Ctrl+Enter)</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="check.qrc">
     <normaloff>:/img/send_to_process.png</normaloff>:/img/send_to_process.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tbClose">
   <property name="geometry">
    <rect>
     <x>431</x>
     <y>241</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Close (Esc)</string>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>30</y>
     <width>230</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="tbDelete">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Delete Selected</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="check.qrc">
     <normaloff>:/img/close.png</normaloff>:/img/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </widget>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete current apk</string>
   </property>
  </action>
  <zorder>textEdit</zorder>
  <zorder>listWidget</zorder>
  <zorder>lMissed</zorder>
  <zorder>lDouble</zorder>
  <zorder>lUnused</zorder>
  <zorder>lLoading</zorder>
  <zorder>lCantD</zorder>
  <zorder>lCantU</zorder>
  <zorder>tbRefresh</zorder>
  <zorder>toolButton_2</zorder>
  <zorder>tbClose</zorder>
  <zorder>listWidget_2</zorder>
  <zorder>lCantM</zorder>
  <zorder>tbDelete</zorder>
 </widget>
 <resources>
  <include location="check.qrc"/>
 </resources>
 <connections/>
</ui>
